sources/Renderer/Direct3D11/Texture/D3D11Texture.cpp

```diff
@@ -296,12 +297,17 @@ HRESULT D3D11Texture::UpdateSubresource(
     const char* srcData = static_cast<const char*>(imageView.data);
     LLGL_ASSERT_PTR(srcData);
 
+    std::uint32_t srcRowStride      = imageView.rowStride > 0 ? imageView.rowStride                 : dataLayout.rowStride;
```

Comment:
Would there ever be a reason to have `imageView.rowStride` be non-zero but smaller than `dataLayout.rowStride`? If not, maybe the maximum between the two values is a reasonable alternative. Let me know what you think, I don't have a strong opinion on this one.